waspc/examples/todoApp/src/client/App.jsx

```diff
@@ -1,10 +1,35 @@
+import logout from "@wasp/auth/logout.js";
+import useAuth from "@wasp/auth/useAuth.js";
+
+import "./Main.css";
+
 export function App({ children }) {
+    const { data: user } = useAuth();
+
     return (
-        <div className="p-6">
-            <header className="mb-6">
-                <h1 className="text-3xl font-bold">ToDo App</h1>
+        <div className="app border-spacing-2 p-4">
+            <header className="flex justify-between">
+                <h1 className="font-bold text-3xl mb-5">
+                    <a href="/">ToDo App</a>
```

Comment:
There is an issue with the usage of `rootComponent` and navigation around the app. 

We are rendering it like this:
```
<App>{router}</App>
```
This means that the `App` component is outside the `<Router>...</Router>` which prevents us from using `<Link />` component in the `rootComponent` which is the proper way to navigate around the app which prevent full page reload.

Related issue: https://github.com/wasp-lang/wasp/issues/1015